### DISCUSSION TITLE
Hilfe gesucht: Massenkorrektur Stundenlohn via GUI

### DISCUSSION THREAD

**Discussion by HeinzWuert** [kimai/kimai#3695] [Questions & Answers]
Rückwirkend zum 01.12. müssen bestimmte bereits erfolgte Buchungen geändert werden.
Dabei soll im Dez nach dem Stundenlohn xx gefiltert und alle in der Anzeige befindlichen Datensätze dann über die vorhandene Massenauswahl mit einem neuen Stundenlohn korrigiert werden.
Es geht hier also nur darum wie filtere ich dieses Datensätze?

**Comment by kevinpapst** [comment]
There is no filter for hourly rate. 
If you can not pin-point it to certain user / projects / customer then I am afraid, but you have to use the database directly.

I am exclusively speaking english here, as it is the universal language for best inclusion of everyone. Posting german means most users  will not understand anything and not benefit from the answer.

**Comment by HeinzWuert** [comment]
Thanks for the answer.
There is an option to filter for a metafield.
meta.project.hilfsform could be used to filter the bookings. Is that possible? Syntax?

**Reply by kevinpapst** [reply]
Docs... https://www.kimai.org/documentation/timesheet.html#filter-and-search

**Comment by HeinzWuert** [comment]
unfortunately the metafield is project and not activity
That means there is no way to filter for december all entrys with entry.rate_nc=x?

**Reply by kevinpapst** [reply]
the search works in every screen with metafields

1. filter all projects with that metafield value
2. search for all of these projects in december timesheets

**Comment by HeinzWuert** [comment]
Thanks a lot, this works!
Didn't realize, that these filters are working additive.

**Reply by kevinpapst** [reply]
complex use-case, but maybe I can add a quicklink from the projects result screen to the timesheet result screen like
- "filter all timesheet for the found projects"
but where would that link be shown?

**Comment by HeinzWuert** [comment]
What about an „additive filter“ function, where both searchfields are displayed, symbol a double magnifying glass for a combined search in projects **and** bookings
